plugins/readme/tsconfig.json

```diff
@@ -0,0 +1 @@
+{ "extends": "../../tsconfig.json" }
```

Comment:
I dont see a tsconfig in the root? 🤔

Reply:
now you don't see it in the package either :D